README.md

```diff
@@ -1,13 +1,13 @@
 # Examples
-Here you can find files and instructions for creating and using example 
+Here you can find files and instructions for creating and using example
 containers from the [sylabs.io blog](https://www.sylabs.io/lab-notes/).  
 
 Each example resides in its own subdirectory with its own README.md and definition
 file.
 
 
-All Definition files provided by Sylabs are under the BSD-3 clause license,
-Unless noted in the file themselves. See the LICENSE.md file for more information. 
+All definition files provided by Sylabs are under the BSD-3 clause license,
+Unless noted in the file themselves. See the LICENSE.md file for more information.
```

Comment:
change "Unless" to "unless" as the previous line ended in a comma, not a period.